api/src/middleware/authenticate.ts

```diff
@@ -76,6 +94,9 @@ const authenticate: RequestHandler = asyncHandler(async (req, res, next) => {
 		req.accountability.role = user.role;
 		req.accountability.admin = user.admin_access === true || user.admin_access == 1;
 		req.accountability.app = user.app_access === true || user.app_access == 1;
+		for (const column of userDynamicVars) {
+			req.accountability.userDynamicVars[column] = user[column];
+		}
```

Comment:
Wondering if we should combine this under a "userData" key together with the users ID 🤔

Reply:
Somehow your comment reminded me of image file name combined with its transformation hash.
It got me thinking about the key used when storing in cache (if implemented).
Something along the lines of `{{user_id}}-{{column_name}}`. 😁